mapie/estimator/interface.py

```diff
@@ -0,0 +1,340 @@
+from __future__ import annotations
+
+from typing import Optional, Tuple, Union
+
+from sklearn.base import RegressorMixin
+
+from mapie._typing import ArrayLike, NDArray
+
+
+class EnsembleEstimator(RegressorMixin):
+    """
+    This class implements methods to handle the training and usage of the
+    estimator. This estimator can be unique or composed by cross validated
+    estimators.
+
+    Parameters
```

Comment:
ERR: it is an interface. There is therefore no `__init__` method. So there's no reason to detail the parameters here.

Reply:
Done